server/grpc_service.go

```diff
@@ -1907,13 +1919,34 @@ func (s *GrpcServer) StoreGlobalConfig(_ context.Context, request *pdpb.StoreGlo
 // LoadGlobalConfig load global config from etcd
 func (s *GrpcServer) LoadGlobalConfig(ctx context.Context, request *pdpb.LoadGlobalConfigRequest) (*pdpb.LoadGlobalConfigResponse, error) {
 	configPath := request.GetConfigPath()
+	if configPath == "" {
+		configPath = globalConfigPath
+	}
+	// `TiKV` cannot use `Names` field Since item value needs to support marshal of different struct types,
+	// it should be set to bytes instead of string.
+	// But for CDC compatibility, we need to keep the Value field.
+	if request.Names != nil {
```

Comment:
What if the `Names` is not nil but empty?

```suggestion
	if len(request.GetNames()) != 0 {
```

Reply:
I don't know what case will names be []{""}? In this case, why not just pass nil🤔